### ISSUE TITLE
Multiprocess

### ISSUE THREAD

**Issue by racacax** [chrippa/livestreamer#1535]
Hello,
To record streams, I use livestreamer. But now, my program is able to record 1,2,3 ... streams in the same time. But when I start 2 processes of livestreamer, the recordings stop one minute after it starts (but when there is only one, it stops when I want it to stop)

Can you correct this ?

PS : With ffmpeg I can start 2,3, 4 ... processes and the recordings don't stop. But livestreamer is better because there is HDS and some streams don't work with ffmpeg.

**Comment by scottbernstein-zz** [comment]
I have successfully run 5 livestreamer processes simultaneously capturing their streams to disk on my Windows 10 64 bit machine without issue.

**Comment by racacax** [comment]
Yes, I've tried this with cmd and that works so I don't know where is the problem in my program. It's aleatory crashing. So It's very strange. I will see if that's my connection.

**Comment by racacax** [comment]
There's something wrong with my program. I will see (ffmpeg is doing the same now)

**Comment by scottbernstein-zz** [comment]
Yep, seems like you can close this issue to me
